ci/vm/provision2.sh

```diff
@@ -59,11 +39,35 @@ xpmem_load() {
   fi
 }
 
+install_gcc() {
+  # Find the GCC ver used to compile the running kernel
+  GCC_VER=$(awk -F ' ' '{print $7}' /proc/version | cut -d'-' -f1)
+  echo "INFO: Installing GCC ver $GCC_VER"
+  wget -q https://ftp.gnu.org/gnu/gcc/gcc-"$GCC_VER"/gcc-"$GCC_VER".tar.xz
+  wget -q https://ftp.gnu.org/gnu/gcc/gcc-"$GCC_VER"/gcc-"$GCC_VER".tar.xz.sig
+  wget -q https://ftp.gnu.org/gnu/gnu-keyring.gpg
+  gpgv --keyring ./gnu-keyring.gpg gcc-"$GCC_VER".tar.xz.sig gcc-"$GCC_VER".tar.xz
+  tar xf gcc-"$GCC_VER".tar.xz
+  (cd gcc-"$GCC_VER" && ./contrib/download_prerequisites)
```

Comment:
Why do you need a subshell here?

Reply:
OK, but pay attention to specificities about sub-shell handling (exit code back propagation, etc.).
Just make sure that this line works as intended in case of sub-shell failure.